features/admin.groups.v1/components/wizard/group-basics.tsx

```diff
@@ -230,26 +222,24 @@ export const GroupBasics: FunctionComponent<GroupBasicProps> = (props: GroupBasi
                 {
                     groupConfig?.allowGroupAddForRemoteUserstores && (
                         <GridRow>
-                            <RootOnlyComponent>
-                                <GridColumn mobile={ 16 } tablet={ 16 } computer={ 10 }>
-                                    <Field
-                                        data-componentid={ `${ componentId }-domain-dropdown` }
-                                        type="dropdown"
-                                        label={ t("roles:addRoleWizard.forms.roleBasicDetails." +
-                                            "domain.label.group") }
-                                        name="domain"
-                                        children={ userStoreOptions }
-                                        placeholder={ t("roles:addRoleWizard." +
-                                            "forms.roleBasicDetails.domain.placeholder") }
-                                        requiredErrorMessage={ t("roles:addRoleWizard.forms." +
-                                            "roleBasicDetails.domain.validation.empty.group") }
-                                        required={ true }
-                                        element={ <div></div> }
-                                        listen={ handleDomainChange }
-                                        value={ initialValues?.basicDetails?.domain ?? userStoreOptions[ 0 ]?.value }
-                                    />
-                                </GridColumn>
-                            </RootOnlyComponent>
+                            <GridColumn mobile={ 16 } tablet={ 16 } computer={ 10 }>
+                                <Field
+                                    data-componentid={ `${ componentId }-domain-dropdown` }
+                                    type="dropdown"
+                                    label={ t("roles:addRoleWizard.forms.roleBasicDetails." +
+                                        "domain.label.group") }
+                                    name="domain"
+                                    children={ userStoreOptions }
+                                    placeholder={ t("roles:addRoleWizard." +
+                                        "forms.roleBasicDetails.domain.placeholder") }
+                                    requiredErrorMessage={ t("roles:addRoleWizard.forms." +
+                                        "roleBasicDetails.domain.validation.empty.group") }
+                                    required={ true }
+                                    element={ <div></div> }
```

Comment:
Is this intentional?

Reply:
This is an existing code block. I removed the enclosing `<RootOnlyComponent>`